tests/stubs/cloud/controllerstub/module.go

```diff
@@ -0,0 +1,81 @@
+package controllerstub
+
+import (
+	"net/http"
+
+	"github.com/kubeedge/beehive/pkg/common/log"
+	"github.com/kubeedge/beehive/pkg/core"
+	"github.com/kubeedge/beehive/pkg/core/context"
+	"github.com/kubeedge/kubeedge/tests/stubs/common/constants"
+)
+
+// Init module
+func init() {
+	core.Register(&ControllerStub{})
+}
+
+// HandlerStub definition
+type ControllerStub struct {
+	context  *context.Context
+	stopChan chan bool
+}
+
+// Return module name
+func (*ControllerStub) Name() string {
+	return constants.ControllerStub
+}
+
+// Return module group
+func (*ControllerStub) Group() string {
+	return constants.ControllerGroup
+}
+
+// Start controller hub
+func (cs *ControllerStub) Start(c *context.Context) {
+	cs.context = c
+	cs.stopChan = make(chan bool)
+
+	// New pod manager
+	pm, err := NewPodManager()
+	if err != nil {
+		log.LOGGER.Errorf("Failed to create pod manager with error: %v", err)
+		return
+	}
+
+	// Start downstream controller
+	log.LOGGER.Infof("Begin to start controllers")
+	downstream, err := NewDownstreamController(cs.context, pm)
+	if err != nil {
+		log.LOGGER.Errorf("New downstream controller failed with error: %v", err)
+		return
+	}
+	downstream.Start()
+
+	// Start upstream controller
+	upstream, err := NewUpstreamController(cs.context, pm)
+	if err != nil {
+		log.LOGGER.Errorf("New upstream controller failed with error: %v", err)
+		return
+	}
+	upstream.Start()
+	log.LOGGER.Infof("End to start controllers")
+
+	// Start http server
+	log.LOGGER.Info("Begin to start http service")
```

Comment:
are these begin and end log needed ?

Reply:
@rohitsardesai83  Thanks. I will remove them